### DISCUSSION TITLE
You have very beautiful editor! Please send your color scheme

### DISCUSSION THREAD

**Discussion by redlom-04** [CleanCut/ultimate_rust_crash_course#17] [Q&A] [answered]
You have very beautiful editor!
Please send your color scheme.
Rust Intellij-idea

**Comment by CleanCut** [comment, marked as answer]
Thanks, I worked hard on customizing the color scheme! ❤️ 

[Here is an export](https://github.com/CleanCut/ultimate_rust_crash_course/files/7679246/IntellijColorSettings.zip) of the settings in the screenshot below:

![image](https://user-images.githubusercontent.com/5838512/145291317-ed507923-fd1f-4d28-95a7-15050121a874.png)
